src/actions/websocket.js

```diff
@@ -239,9 +253,6 @@ function handleEvent(msg) {
     case WebsocketEvents.UPDATE_TEAM:
         doDispatch(handleUpdateTeamEvent(msg));
         break;
-    case WebsocketEvents.PATCH_TEAM:
```

Comment:
Why was this removed?

Reply:
I discussed it with @jespino before in a private conversation, sorry I did not write a comment in PR. Long story short, there is no such event 🙂. By introducing Flow, the flow checker showed that `WebsocketEvents.PATCH_TEAM` constant is not existing. I then checked with @jespino and he confirmed that it is not there and there is also no `PATCHED_TEAM` on the BackEnd. So we decided that it would be the best to delete the case and the handle function.

Reply:
@reflog sorry I am not familiar with the way it is handled correctly, do I get to click on the `Resolve conversation` button? 🙂

Reply:
Thanks for the explanation @mzaks !
About resolve conversation - the originator of the comment presses Resolve if the question was answered (so in this case I will press Resolve) or by the committer if they handled the comment (for example if you fixed the bug mentioned).